mail_optional_autofollow/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please remove everywhere utf-8 line as in v11 it is not needed.

Reply:
done